torchrl/data/replay_buffers/samplers.py

```diff
@@ -1079,6 +1079,197 @@ def load_state_dict(self, state_dict: Dict[str, Any]) -> None:
         return SamplerWithoutReplacement.load_state_dict(self, state_dict)
 
 
+class PrioritizedSliceSampler(SliceSampler, PrioritizedSampler):
```

Comment:
Note: Ideally I would like to avoid multi-inheritance, since it is prone to generate spaghetti code, but this pattern was used elsewhere in the code so I kept the same logic: `SliceSamplerWithoutReplacement(SliceSampler, SamplerWithoutReplacement)`

Reply:
I agree I don't like it either. The other option is to copy paste methods, or port them out of the classes and make the classes rely on external functions. 
To me if it seems organic it's ok, even if it isn't the best feature in python